samples/bluetooth/peripheral_uart/src/main.c

```diff
@@ -81,7 +81,7 @@ static const struct bt_data sd[] = {
 	BT_DATA_BYTES(BT_DATA_UUID128_ALL, BT_UUID_NUS_VAL),
 };
 
-#if CONFIG_BT_NUS_UART_ASYNC_ADAPTER
+#if CONFIG_UART_ASYNC_ADAPTER
 UART_ASYNC_ADAPTER_INST_DEFINE(async_adapter);
 #else
 static const struct device *const async_adapter;
```

Comment:
It's a bit weird to define a variable just to be able to compile a piece of code that will not be used anyway and is supposed to be optimized out.

Reply:
Would it look better if I define it to NULL? I just want to use as small amount of preprocessor conditionals as possible.